roachtest: follower-reads/nodes=3 failed

SHA: https://github.com/cockroachdb/cockroach/commits/e4127466ae0a44efb624afcaf0861903028b6e18

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=follower-reads/nodes=3 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1129825&tab=buildLog

```
The test failed on release-2.1:
	follower_reads.go:191,test.go:1206: error verifying node values: pq: unknown function: experimental_follower_read_timestamp()

```

--- a/pkg/cmd/roachtest/follower_reads.go
+++ b/pkg/cmd/roachtest/follower_reads.go
@@ -78,6 +78,18 @@ func runFollowerReadsTest(ctx context.Context, t *test, c *cluster) {
 	if _, err := db.ExecContext(ctx, "SET CLUSTER SETTING kv.closed_timestamp.follower_reads_enabled = 'true'"); err != nil {
 		t.Fatalf("failed to enable follower reads: %v", err)
 	}
+	// Disable load based splitting and range merging because splits and merges
+	// interfere with follower reads. Rhis test's workload regularly triggers load
+	// based splitting in the first phase creating small ranges which later
+	// in the test are merged. The merging tends to coincide with the final phase
+	// of the test which attempts to observe low latency reads leading to
+	// flakiness.
+	if _, err := db.ExecContext(ctx, "SET CLUSTER SETTING kv.range_split.by_load_enabled = 'false'"); err != nil {
+		t.Fatalf("failed to disable load based splitting: %v", err)
+	}
+	if _, err := db.ExecContext(ctx, "SET CLUSTER SETTING kv.range_merge.queue_enabled = 'false'"); err != nil {
+		t.Fatalf("failed to disable range merging: %v", err)
+	}
 	if r, err := db.ExecContext(ctx, "CREATE DATABASE test;"); err != nil {
 		t.Fatalf("failed to create database: %v %v", err, r)
 	}

